src/ipc/handler.c

```diff
@@ -700,6 +741,10 @@ static int ipc_glb_pm_message(uint32_t header)
 		return ipc_pm_context_size(header);
 	case SOF_IPC_PM_CORE_ENABLE:
 		return ipc_pm_core_enable(header);
+	case SOF_IPC_PM_STATE_SET:
+		return ipc_pm_state_set(header);
+	case SOF_IPC_PM_STATE_GET:
+		return ipc_pm_state_get(header);
```

Comment:
@lbetlej just for my understanding. When would the driver use the ipc_pm_state_get()? Clearly when the DSP is in D3, this cannot be used.

Reply:
@ranj063 I'm not sure, but with IPCs from #1339 it may be possible, they work in D3 afaik.
 @lbetlej am I right?